plates/dimensions_tests.py

```diff
@@ -0,0 +1,76 @@
+import unittest
+from dimensions import Dimensions
+
+class TestDimensions(unittest.TestCase):
+    """Tests for Dimensions class"""
+
+    def test_row_from_4_x_6_dimension_is_saved(self):
+        dimensions = Dimensions(4, 6)
```

Comment:
We don't usually test simple getters/setters/constructors that just read/write values. Most likely they will be tested implicitly in other tests that are more high-level.

Reply:
Personally I'd only cover `get_capacity()` method